homeassistant/components/brother/config_flow.py

```diff
@@ -22,6 +23,25 @@
         vol.Optional(CONF_TYPE, default="laser"): vol.In(PRINTER_TYPES),
     }
 )
+RECONFIGURE_SCHEMA = vol.Schema({vol.Required(CONF_HOST, default=""): str})
```

Comment:
Why does CONF_HOST have a default value which is the empty string?

Reply:
Good question, a copy-paste mistake.